scanner/updater/enrichment/cvss_test.go

```diff
@@ -0,0 +1,80 @@
+package enrichment
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"io"
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/quay/claircore/enricher/cvss"
+	"github.com/quay/claircore/libvuln/driver"
+	"github.com/quay/zlog"
+)
+
+var (
+	pkgClient = &http.Client{}
+
+	fp driver.Fingerprint
+)
+
+// TestCVSS verifies the accessibility and correct parsing of CVSS data stored in Google Storage.
```

Comment:
I understand your approach. But it's always best to have unit tests that are independent and don't rely on external dependencies. Have you considered using test data instead of Google storage for this particular unit test?

Reply:
Actually what I want to test is to make sure Claircore enricher works with customized URL, which is google storage in our case (ACS scanner). Because eventually the central needs to get the data from Google storage but not the default url.

Reply:
> I want to test is to make sure Claircore enricher works with customized URL

It's common for us to trust that third-party libraries will function as intended. I am not sure if a Unit Test is the right tool to verify ClairCore is behaving as expected. Unit Tests usually are designed to be isolated, so it is deterministic and reproducible.

> Because eventually the central needs to call FetchEnrichment() with Google storage URL but not the default URL.

It seems that this should run in the Matcher, as part of the updater. The data would then be used to enrich a vulnerability report. Is my understanding correct?